ui/admin/app/components/mapping-list/index.hbs

```diff
@@ -0,0 +1,64 @@
+<Hds::Form::Fieldset
+  @layout={{@layout}}
+  @isRequired={{@isRequired}}
+  @isOptional={{@isOptional}}
+  as |F|
+>
+  {{yield F}}
+  <F.Control>
+    <ul class='mapping-list'>
+      {{#each @options as |option index|}}
+        <li>
+          <Hds::Form::TextInput::Field
+            @value={{option.key}}
+            @type='text'
+            disabled={{@disabled}}
+            {{on 'input' (set-from-event option 'key')}}
+          />
+          <Hds::Form::TextInput::Field
+            @value={{option.value}}
+            @type='text'
+            disabled={{@disabled}}
+            {{on 'input' (set-from-event option 'value')}}
+          />
+          <div>
```

Comment:
I needed these wrapper divs around the buttons to provide a container for whitespace so I could left align the different sized buttons.

Happy to hear of a better alternative to accomplish this.

Reply:
Can this have a class name?

Reply:
Sure, but even if there's no associated styles with the class?